Libplanet/Net/Swarm.cs

```diff
@@ -223,6 +224,8 @@ static Swarm()
 
         public AsyncAutoResetEvent TxReceived { get; }
 
+        public AsyncAutoResetEvent BlockAppended { get; }
```

Comment:
This should be added to the changelog.

Reply:
I didn't add the part to the changelog, because another event properties (e.g. `TxReceived`) weren't added, too.
Then they should be added to the changelog, too?